dspace-server-webapp/src/test/java/org/dspace/app/rest/SubscriptionRestRepositoryIT.java

```diff
@@ -1070,7 +1075,7 @@ public void linkedEpersonOfSubscriptionAdminTest() throws Exception {
         String tokenAdmin = getAuthToken(admin.getEmail(), password);
         getClient(tokenAdmin).perform(get("/api/core/subscriptions/" + subscription.getID() + "/eperson"))
                              .andExpect(status().isOk())
-                             .andExpect(jsonPath("$", is(EPersonMatcher.matchEPersonEntry(eperson))));
+                             .andExpect(jsonPath("$", is(EPersonMatcher.matchEPersonOnEmail(eperson.getEmail()))));
```

Comment:
Full entity machers now break these tests

Reply:
Matchers can't always retrieve services from factories, so a simpler matcher is used here as a workaround.
I'll look into the larger problem, but in this particular case we don't really gain much by using the full matcher anyway.

See https://github.com/DSpace/DSpace/pull/9735#discussion_r1766419940